MapboxCoreNavigation/RouteController.swift

```diff
@@ -22,8 +22,8 @@ public protocol RouteControllerDelegate: class {
     @objc(routeController:shouldRerouteFromLocation:)
     optional func routeController(_ routeController: RouteController, shouldRerouteFrom location: CLLocation) -> Bool
     
-    @objc(routeController:shouldIncrementLegWhenArrivingAtWaypoints:)
-    optional func routeController(_ routeController: RouteController, shouldIncrementLegWhenArrivingAtWaypoints waypoint: Waypoint) -> Bool
+    @objc(routeController:shouldIncrementLegWhenArrivingAtWaypoint:)
+    optional func routeController(_ routeController: RouteController, shouldIncrementLegWhenArrivingAtWaypoint waypoint: Waypoint) -> Bool
```

Comment:
The Swift name should be `routeController(_:shouldIncrementLegWhenArrivingAt:)`. “Waypoint” is inferred from the argument’s type.